test/gateway/simple_stake/release_to.js

```diff
@@ -116,7 +116,6 @@ contract('SimpleStake.releaseTo()', (accounts) => {
 
     await Utils.expectRevert(
       simpleStake.releaseTo(beneficiary, releasedAmount, { from: gateway }),
-      'Underflow when subtracting.',
```

Comment:
This is strange that the require error messages are not available.
Was checking the release candidate `v2.3.0-rc.1` of `OpenZeppelin/openzeppelin-solidity` and it looks like the error messages will be available.